e2e/settings/index.spec.ts

```diff
@@ -275,39 +275,36 @@ test("it is possible to edit variables", async ({ page }) => {
   ).toHaveValue(updatedType);
 });
 
-test("it is possible to handle broadcasts", async ({ page }) => {
-  const { broadcast } = await createBroadcasts(namespace);
+test("it is possible to update broadcasts", async ({ page }) => {
+  const { broadcast: broadcasts } = await createBroadcasts(namespace);
+  expect(broadcasts, "test data has been created").toBeTruthy();
+
   await page.goto(`/${namespace}/settings`);
-  expect(broadcast, "broadcast should be truthy").toBeTruthy();
-  //check the initial state
+
+  // check the initial state
   for (let i = 0; i < BroadcastsSchemaKeys.length; i++) {
     const key = BroadcastsSchemaKeys[i] || "directory.create";
     const checkbox = page.getByTestId(`check.${key}`);
     const isChecked = await checkbox.isChecked();
-    expect(
-      isChecked,
-      `${key}-checkbox status should be the same as in the broadcast`
-    ).toBe(broadcast[key]);
+    expect(isChecked, `checkbox for ${key} has the expected value`).toBe(
```

Comment:
Again, I edited the description to be a bit more general.

"the same as in the broadcast" is only clear if you read the code and know how the variables are named. 

The test log should be easy to understand without knowing the code.